pkg/controllers/dynakube/injection/conditions.go

```diff
@@ -32,3 +46,80 @@ func setCodeModulesInjectionCreatedCondition(conditions *[]metav1.Condition) {
 	}
 	_ = meta.SetStatusCondition(conditions, condition)
 }
+
+func setNamespacesMonitoredSelectorCondition(conditions *[]metav1.Condition, selectorType string, configured bool, names []string) {
```

Comment:
why is this not done in the mapper?

Reply:
I tried to group it at the beginning of the story - but now I see this does not make any sense https://github.com/Dynatrace/dynatrace-operator/pull/5484/commits/f3111d79d5e3ffe94af64b01027b8ac1f6ef5ce3